MapboxNavigation/RouteMapViewController.swift

```diff
@@ -248,19 +217,28 @@ class RouteMapViewController: UIViewController {
         self.mapView.setContentInset(self.contentInsets, animated: true, completionHandler: nil)
         self.mapView.setNeedsUpdateConstraints()
     }
+	
+    // MARK: - UIContentContainer
+	
+    override func preferredContentSizeDidChange(forChildContentContainer container: UIContentContainer) {
```

Comment:
Is this a new implementation or was it just moved?

Reply:
just moved